docker-compose.yml

```diff
@@ -50,6 +50,8 @@ services:
     expose: ["5000"]
     networks:
      - internal
+    entrypoint: ""
+    command: "./runserver.sh"
```

Comment:
Mostly for local development, not needing to exec into a container to run things like `initdb`. It also "feels" more docker-y, since that's what entrypoint is for, it's disappointing to not be able to use it.